app/views/layouts/_head.html.erb

```diff
@@ -4,6 +4,7 @@
   <%= csp_meta_tag %>
 
   <meta name="viewport" content="width=device-width, initial-scale=1" />
+  <script src='https://api.mapbox.com/mapbox-gl-js/v1.12.0/mapbox-gl.js'></script>
```

Comment:
There are obviously tradeoffs in both directions, but there's also the option of packaging this up with webpack:
https://docs.mapbox.com/mapbox-gl-js/api/ and https://docs.mapbox.com/mapbox-gl-js/api/#mapbox-css.

I'm good with using the cdn for now.

Reply:
we can circle back to this. we first tried adding it via webpack but we encountered problems.